src/app/modules/admin-core/components/bread-crum/bread-crum.component.html

```diff
@@ -1,5 +1,6 @@
-<div *ngIf="edit != 'edit/:id'" class="pageName">
-    <span *ngFor="let data of crumData?.title">
+<div *ngIf ="show" class="pageName">
+    <span *ngFor="let data of crumData?.title; let i = index">
```

Comment:
search for last in ngFor

Reply:
yes i changed to last, but is it makes any difference ?